src/Glob.php

```diff
@@ -61,7 +61,7 @@ protected static function systemGlob($pattern, $flags)
                 self::GLOB_NOSORT   => GLOB_NOSORT,
                 self::GLOB_NOCHECK  => GLOB_NOCHECK,
                 self::GLOB_NOESCAPE => GLOB_NOESCAPE,
-                self::GLOB_BRACE    => GLOB_BRACE,
+                self::GLOB_BRACE    => defined('GLOB_BRACE') ? GLOB_BRACE : 0,
```

Comment:
Before posting the issue here, I've checked some approaches taken by other softwares and they did the same and defaulted it to `0`.

http://git.alpinelinux.org/cgit/aports/plain/main/asterisk/musl-glob-compat.patch
http://lists.alpinelinux.org/alpine-aports/1379.html